plugin/docker/containers.go

```diff
@@ -25,8 +26,14 @@ func (cs *containers) List(ctx context.Context) ([]plugin.Entry, error) {
 	activity.Record(ctx, "Listing %v containers in %v", len(containers), cs)
 	keys := make([]plugin.Entry, len(containers))
 	for i, inst := range containers {
+		name := inst.ID
+		if len(inst.Names) > 0 {
+			// Names must currently be unique in the docker runtime.
```

Comment:
I'm not sure how this comment clarifies the `TrimPrefix` call here. A bit more explanation would be useful.

Reply:
Oh, that was why I didn't append the ID. I'll clarify the comment.